backend/layers/business/entities.py

```diff
@@ -1,6 +1,6 @@
 from typing import List, Optional
 
-from pydantic.dataclasses import dataclass
+from dataclasses import dataclass
```

Comment:
This is necessary for now since `pydantic` isn't declared in the dependencies. We should do that in a separate PR.